lib/components/user/with-logged-in-user-support.js

```diff
@@ -37,27 +37,34 @@ export default function withLoggedInUserSupport (WrappedComponent, requireLogged
  * Upon completion (or if no user is logged in or if auth is disabled), it renders children.
  */
 class UserLoaderScreen extends Component {
-  componentDidUpdate () {
-    const { auth, fetchOrInitializeUser, loggedInUser } = this.props
+  /**
+   * Determines whether user data should be fetched.
+   * @returns true if the logged-in user has passed Auth0 authentication and state.user.loggedInUser has not been set; false otherwise.
+   */
+  shouldBeFetchingUser = () => {
```

Comment:
Please rename to `shouldFetchUser` since it seems this shouldn't be a continuous action that is performed.

Reply:
Done. (I picked that name because it read better in the render method, but that was a split call)